ngraph/test/visitors/op/binary_elementwise.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+
+#include "ngraph/op/util/attr_types.hpp"
+#include "util/visitor.hpp"
+
+using namespace ngraph;
```

Comment:
Don't use `using namespace` in header in global namespace.

Reply:
Fixed, thank you!